test/utils/test_print.py

```diff
@@ -0,0 +1,40 @@
+import pytest
+import torch
+
+from kornia.utils import image_to_string, print_image
+
+
+class TestImageToString:
+    def test_value(self):
+        image = torch.arange(16).reshape(1, 4, 4).repeat(3, 1, 1).long() * 16
+        out = image_to_string(image)
+
+        expected = (
+            "\033[48;5;16m  \033[48;5;16m  \033[48;5;16m  \033[48;5;59m  \033[0m\n"
+            "\033[48;5;59m  \033[48;5;59m  \033[48;5;59m  \033[48;5;59m  \033[0m\n"
+            "\033[48;5;102m  \033[48;5;102m  \033[48;5;145m  \033[48;5;145m  \033[0m\n"
+            "\033[48;5;145m  \033[48;5;188m  \033[48;5;188m  \033[48;5;231m  \033[0m\n"
+        )
+        assert out == expected
+
+    def test_errors(self):
```

Comment:
to match with the codebase
```suggestion
    def test_exception(self):
```